blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/IMetricSerializerTest.java

```diff
@@ -73,7 +73,6 @@ public void testSetSerialization() throws IOException {
 
         BluefloodSetRollup setDeserialized = mapper.readValue(setValue, BluefloodSetRollup.class);
         String setSerialized = mapper.writeValueAsString(setDeserialized);
-        Assert.assertEquals(setValue, setSerialized);
```

Comment:
Nevermind you explained it in the PR description.  Thanks for catching @shintasmith !